reference/7.0/Microsoft.PowerShell.Utility/Invoke-WebRequest.md

```diff
@@ -1182,12 +1185,32 @@ You can pipe the body of a web request to `Invoke-WebRequest`.
 
 Beginning with PowerShell 6.0.0 `Invoke-WebRequest` supports basic parsing only.
 
-For more information about the **Microsoft.PowerShell.Commands.BasicHtmlWebResponseObject** object
-type, see
+For more information, see
 [BasicHtmlWebResponseObject](/dotnet/api/microsoft.powershell.commands.basichtmlwebresponseobject).
 
-For more information about how .NET provides proxy services to PowerShell, see
-[Accessing the Internet Through a Proxy](/dotnet/framework/network-programming/accessing-the-internet-through-a-proxy).
+Because of changes in .NET Core 3.1, PowerShell 7.0 and higher supports proxy configuration defined
+by the following environment variables:
+
+- `$env:http_proxy` or `$env:HTTP_PROXY`
+
+  Set this variable to the endpoint of an HTTP proxy. For example: `http://127.0.0.1:3001`
+
+- `$env:https_proxy` or `$env:HTTPS_PROXY`
+
+  Set this variable to the endpoint of an HTTPS proxy. For example: `https://127.0.0.1:8001`
+
+- `$env:all_proxy` or `$env:ALL_PROXY`
+
+  Set this variable to the endpoint to be used for any protocol. For example:
+  `socks://127.0.0.1:8081`.
+
+- `$env:no_proxy` or `$env:NO_PROXY`
+
+  Set this variable to a comma-separated list of host names that should not use a proxy when being
+  accessed. For example: `*.test.example.com,.example2.com`.
```

Comment:
I was under the impression, and a brief experience seemed to validate, that wildcards do not work in this variable the way they do in the OS internet settings.